output/manager.go

```diff
@@ -1,10 +1,16 @@
 package output
 
 import (
+	"sync"
+	"time"
+
 	"github.com/sirupsen/logrus"
 	"go.k6.io/k6/metrics"
 )
 
+// TODO: completely get rid of this, see https://github.com/grafana/k6/issues/2430
+const sendBatchToOutputsRate = 50 * time.Millisecond
+
```

Comment:
What do we plan instead of this? Would we replace it with a pull-based model?

Reply:
not sure, maybe :thinking: we'll certainly need to do some benchmarking before, maybe the current solution is the most efficient one, who knows... :thinking:

Reply:
I linked the code where we mention this in the issue's description